README.md

```diff
@@ -143,6 +164,44 @@ $ GOOS=linux make docker.push
 
 **NOTE:** Set HUB and TAG per your docker registry.
 
+### Helm
+
+The helm package tarfile can be created via
+
+```
+helm package $GOPATH/src/istio.io/deployments/kubernetes/install/helm/istio-cni
+```
+
+#### Serve Helm Repo
+
+An example for hosting a test repo for the helm istio-cni package:
+1. Create package tarfile with `helm package $GOPATH/src/istio.io/deployments/kubernetes/install/helm/istio-cni`
+1. copy tarfile to dir to serve the repo from
+1. Run `helm serve --repo-path <dir where helm tarfile is>`
+   1. The repo URL will be output (`http://127.0.0.1:8879`)
+
+To use this repo via `helm install`:
+1. `helm repo add local_istio http://127.0.0.1:8879`
+1. `helm repo update`
```

Comment:
It's preferable to put these longer commands in a code block instead. Also, mark with \```sh wherever needed.